frontend/views/containers/sidebar/Navigation.vue

```diff
@@ -135,6 +124,10 @@ export default {
     logo () {
       const name = this.colors.theme === 'dark' ? '-white' : ''
       return `/assets/images/logo-transparent${name}.png`
+    },
+    isInert () {
+      // TODO: Update window.innerWidth on resize.
+      return !this.ephemeral.isActive && window.innerWidth < 1200
```

Comment:
This exists twice, so it needs to be DRY'd somehow, perhaps a mixin or something

Reply:
It exists twice but I don't think it's a motive to create an abstraction for something so raw. It's the same as saying "margin-right: 1rem" or `<input type="email />` exists twice, we should create a CSS class / component for it.

Once we solve the "js resize/responsive" thing, we can find a better solution. For now, I would leave it as it is.

ref: [AHA programming](https://kentcdodds.com/blog/aha-programming)

Reply:
The specific concern here is the part where it checks the innerWidth against a constant value. That's not OK, since it means we have to remember to keep the constant value (1200) the same in both locations. Even in our CSS classes we make sure to use variables for such situations to avoid inconsistencies arising.

Reply:
I think it's perfectly fine to have this line repeated. But If greg is concerned, about the constant, we can still extract that constant only and replace with a settings variable.

Reply:
I searched for all raw values related to breakpoints and replaced them with constants.